pkg/tsdb/cloudwatch/log_query.go

```diff
@@ -8,13 +8,30 @@ import (
 )
 
 func logsResultsToDataframes(response *cloudwatchlogs.GetQueryResultsOutput) (*data.Frame, error) {
+	nonEmptyRows := make([][]*cloudwatchlogs.ResultField, 0)
+	// Sometimes cloud watch can send empty rows
+	for _, row := range response.Results {
+		if len(row) == 0 {
+			continue
+		}
+		if len(row) == 1 {
+			// Sometimes it sends row with only timestamp
+			if _, err := time.Parse(cloudWatchTSFormat, *row[0].Value); err == nil {
```

Comment:
Do we know the row.Value can't be nil, if can be, then dereferencing it here will panic.

Example: https://play.golang.org/p/239TWNUyB_J

Reply:
Yeah I think this would be really weird on cloud watch part. If it sends only one field in the rows it should have some value. But can add check for that just for sure.